service/task.go

```diff
@@ -418,6 +434,23 @@ func (uis *UIServer) taskPage(w http.ResponseWriter, r *http.Request) {
 	}{uiTask, taskHost, pluginContent, uis.Settings.Jira.Host, permissions, uis.GetCommonViewData(w, r, false, true)}, "base", "task.html", "base_angular.html", "menu.html")
 }
 
+func getAbortedBy(abortedByTaskId string) (*abortedByDisplay, error) {
+	abortedTask, err := task.FindOne(task.ById(abortedByTaskId))
+	if err != nil || abortedTask == nil {
+		return nil, errors.Wrap(err, "problem getting abortedBy task")
```

Comment:
I just realized, in the case where abortedTask is nil we'll be returning `nil, nil` since `errors.Wrap(err, ...)` returns nil when err is nil.
We probably need to make a new error in that case.

Reply:
good point! fixed.